commercetools-sdk-base/src/main/java/io/sphere/sdk/search/model/FilteredFacetExpressionImpl.java

```diff
@@ -25,4 +25,9 @@ public boolean equals(Object o) {
     public String alias() {
         return super.alias();
     }
+
+    @Override
```

Comment:
I followed the alias example here. Maybe  im missing something but what is the sense of overriding a method to call `super.method` ?